test/apps/test_ucx_tls.py

```diff
@@ -185,12 +206,25 @@ def test_ucx_tls_positive(tls):
     print("Found TL doesn't belong to the allowed UCX_TLS")
     sys.exit(1)
 
-def test_ucx_tls_negative(tls):
-    # Use TLS list in "negate" mode and verify that the found tl is not in the list
-    found_tl = find_am_transport(None, tls="^"+tls)
-    print(f"Using UCX_TLS={tls}, found TL: {found_tl}")
+def test_ucx_tls_negative(tls, protocol="am", forbidden_tls=None):
+    # Use TLS list in "negate" mode
+    found_tl = find_transport(tls="^"+tls, protocol=protocol)
+    print(f"Using UCX_TLS=^{tls}, found {protocol} TL: {found_tl}")
+    if not found_tl:
+        print("No available TL found")
+        sys.exit(1)
+
+    # If forbidden_tls is provided, verify that the found tl is not in that list
+    if forbidden_tls is not None:
+        for tl in forbidden_tls:
+            if found_tl == tl:
+                print("No available TL found")
```

Comment:
Seems this error message should be adjusted, like "found forbidden TL {found_tl}
also, can we check directly if "found_tl in forbidden_tls" ?